src/sdk/v2/sdk.store.ts

```diff
@@ -183,6 +183,12 @@ export const _store_initialize = async (params: InitParams) => {
   const account = await signer?.getAddress();
   if (account != null && signer != null) {
     _sdkStore.setState({ signerInitialized: true, account, signer });
+  } else {
+    _sdkStore.setState({
+      signerInitialized: false,
+      account: undefined,
+      signer: undefined,
+    });
```

Comment:
This fixes a bug where switching between accounts leaves `fhenixsdk` in a state where it believes that there is a valid signer, but really there isn't